specification/edgeorder/EdgeOrder.Management/AddressResource.tsp

```diff
@@ -0,0 +1,144 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.EdgeOrder;
+/**
+ * Address Resource.
+ */
+model AddressResource
+  is Azure.ResourceManager.TrackedResource<AddressProperties, false> {
+  ...ResourceNameParameter<
+    Resource = AddressResource,
+    KeyName = "addressName",
+    SegmentName = "addresses",
+    NamePattern = "^[-\\w\\.]+$"
+  >;
+}
+
+@armResourceOperations
+interface AddressResources {
+  /**
+   * Get information about the specified address.
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-openapi" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
```

Comment:
>  #suppress "@azure-tools/typespec-azure-core/no-openapi" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"

Is it fixable? When will it be fixable?

Reply:
This should be handled by https://github.com/Azure/typespec-azure/issues/2683. We will clean up later.